docs/guides/securing-apps/partials/overview/getting-started.adoc

```diff
@@ -19,7 +19,7 @@ endif::[]
 * <@links.securingapps id="javascript-adapter"/>
 
 ==== Node.js (server-side)
-* <@links.securingapps id="nodejs-adapter"/>
+* [https://github.com/keycloak/keycloak/blob/main/docs/guides/securing-apps/nodejs-adapter.adoc][Keycloak Node.js adapter]
```

Comment:
Same like above, properly can be replace with proper link (Specifically for this one, it is definitely not correct as it refers to the file deleted in this PR, so I suppose the link will go to 404 after the PR is merged).

Reply:
Update: the proper link should be: https://www.keycloak.org/securing-apps/nodejs-adapter (in the community)